javascript/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@parity/zombienet",
-  "version": "1.3.10",
+  "name": "zombienet",
+  "version": "0.1.0",
```

Comment:
why we are changing this?

Reply:
No need to keep the same name and version as CLI. Its the root of repo and we are going to update just that.